pkg/node/addresstype.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Authors of Cilium
```

Comment:
This should be somewhere in `pkg/k8s`

Reply:
I'm ambivalent as to where it goes. I'm guessing your rationale is that since it comes from K8s we should put it in `pkg/k8s`? I like it here because an address type belongs to a node.

Reply:
> I'm guessing your rationale is that since it comes from K8s we should put it in pkg/k8s? 

yes, or it should be under `pkg/addressing` or `pkg/address`. conversation continues in [1]

Reply:
What about `pkg/node/addressing`? It is a separate package, but delineates that the address types in there are specifically for nodes. I think this meets what I desire (collocating it with node in some way) with your desire for a separate package.

Reply:
I will add `pkg/node/addressing` in the next upload.